.github/workflows/run-tests.yml

```diff
@@ -6,18 +6,28 @@ name: Run Tests
 on: [push, pull_request]
 
 jobs:
+  call_run_tests-react-18:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/run_tests.yml@v1
+    with:
+      node_matrix: '["16.x", "18.x"]'
+    secrets:
+      MAPBOX_API_KEY: ${{ secrets.MAPBOX_API_KEY }}
   call_run_tests-react-17:
     uses: yext/slapshot-reusable-workflows/.github/workflows/run_tests.yml@v1
     with:
+      build_script: |
+        npm i -D react@17 react-dom@17
```

Comment:
you'll have to do the `@testing-library/react-hooks` install in this step so there aren't peer dependency conflicts (e.g. something like `npm i -D react@17 react-dom@17 @testing-library/react-hooks`). also you should choose a specific major version of `@testing-library/react-hooks` to pin to so that we don't get any breaking changes if they release a new major version of the package